domains/zeto/integration-test/e2e_fungible_test.go

```diff
@@ -208,14 +210,92 @@ func (s *fungibleTestSuiteHelper) testZeto(t *testing.T, tokenName string, useBa
 	require.NoError(t, err)
 
 	expectedCoins += 2 // the deposit call produces 2 output UTXOs for the receiver
-	coins = findAvailableCoins(t, ctx, s.rpc, s.domain, zetoAddress, nil, isNullifiersToken)
+	coins = findAvailableCoins(t, ctx, s.rpc, s.domain, zetoAddress, isNullifiersToken)
 	require.Len(t, coins, expectedCoins)
 
 	log.L(ctx).Info("*************************************")
 	log.L(ctx).Infof("Withdraw back to ERC20 balance from Zeto")
 	log.L(ctx).Info("*************************************")
 	_, err = s.withdraw(ctx, zetoAddress, controllerName, 100)
 	require.NoError(t, err)
+
+	if tokenName != constants.TOKEN_ANON {
```

Comment:
Might be worth a comment to indicate why this test is different for different tokens.